pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -755,6 +755,7 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
       this.writeFile('/etc/init.d/rancher-desktop-guestagent', SERVICE_GUEST_AGENT_INIT, { permissions: 0o755 }),
       (async() => {
         await this.writeConf('rancher-desktop-guestagent', guestAgentConfig);
+        await this.execCommand('/sbin/rc-update', 'add', 'rancher-desktop-guestagent', 'default');
       })(),
```

Comment:
Any reason not to run this in parallel?
```suggestion
      this.writeConf('rancher-desktop-guestagent', guestAgentConfig),
      this.execCommand('/sbin/rc-update', 'add', 'rancher-desktop-guestagent', 'default'),
```